addons/l10n_rs/models/account_move.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_rs_company_country_id = fields.Many2one(string='Company Country ID', related='company_id.country_id')
```

Comment:
This field should not be necessary, you can use `country_code` instead

Reply:
I looked into that, but I wasn't sure it was doing what I wanted :sweat_smile: 
It seems to be the country used for tax reports.
I just need to know that the active company is Serbian :man_shrugging:

Reply:
Sorry about that last comment, I was actually thinking about `l10n_rs_is_serbian_company` :upside_down_face: 
So Yes, I think I can use `country_code` indeed :+1: